packages/gafl-webapp-service/src/pages/renewals/renewal-inactive/__tests__/route.spec.js

```diff
@@ -221,5 +229,18 @@ describe('renewal-inactive > route', () => {
       expect(result.bodyMessage).toBe('')
       expect(result.title).toBe('')
     })
+
+    it('addLanguageCodeToUri is called with the expected arguments', async () => {
+      mockStatusCacheGet.mockImplementationOnce(() => ({
+        authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate: '2020-12-13T23:59:59Z' },
+        referenceNumber: 'ABC123'
+      }))
```

Comment:
Up to you whether you change this and the other uses of the mockStatusCacheGet as I've asked you to in the other files, but we'll have to do it sooner or later because it's a bad pattern.